### PR TITLE
Added Gamerherz

### PR DESCRIPTION
**Pull request by Nicouschulas**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/gamerherz/manifest.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  "direct_ip": "Gamerherz.eu",
+  "server_wildcards": [
+    "%.gamerherz.eu"
+  ],
+  "supported_languages": [
+     "de",
+     "en"
+  ],
+  "social": {
+    "web": "https://www.gamerherz.eu/",
```
Comment (maaaathis) [inline minecraft_servers/gamerherz/manifest.json:13]: Returns a error 404

File: minecraft_servers/gamerherz/background.png
Comment (maaaathis) [inline minecraft_servers/gamerherz/background.png:1]: Compress

File: minecraft_servers/gamerherz/background@2x.png
Comment (maaaathis) [inline minecraft_servers/gamerherz/background@2x.png:1]: Compress

**Review by maaaathis — CHANGES_REQUESTED** [review]

File: minecraft_servers/gamerherz/manifest.json (hunk 1 above)
Comment (Nicouschulas) [reply minecraft_servers/gamerherz/manifest.json:13]: The Website is in two or three days back online.

File: minecraft_servers/gamerherz/icon@2x.png
Comment (DoJapHD) [inline minecraft_servers/gamerherz/icon@2x.png:1]: Not a @2x version; re-read the "definition" in the docs

File: minecraft_servers/gamerherz/logo@2x.png
Comment (DoJapHD) [inline minecraft_servers/gamerherz/logo@2x.png:1]: Not a @2x version; re-read the "definition" in the docs

File: minecraft_servers/gamerherz/manifest.json (hunk 1 above)
Comment (DoJapHD) [inline minecraft_servers/gamerherz/manifest.json:13]: Website offline

File: minecraft_servers/gamerherz/manifest.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{
+  "server_name": "Gamerherz",
```
Comment (DoJapHD) [inline minecraft_servers/gamerherz/manifest.json:2]: server_name in lowercase

File: minecraft_servers/gamerherz/manifest.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  ],
+  "social": {
+    "web": "https://www.gamerherz.eu/",
+    "twitter": "Gamerherz.eu",
+    "tiktok": "Gamerherz.eu",
+    "discord": "https://discord.gg/Z3rrqYSpUE",
+    "youtube": "https://www.youtube.com/channel/UCoPAhhSl7Y3sek7qXkQCqWw",
+  },
+  "gamemodes": {
+    "Paintball": {
```
Comment (DoJapHD) [inline minecraft_servers/gamerherz/manifest.json:20]: JSON key in lowercase

File: minecraft_servers/gamerherz/manifest.json [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+      "name": "Paintball", 
+      "command": "/quickjoin Paintball-1",
+      "versions": "1.8<*"
+    },
+    "Duels 1.8": {
```
Comment (DoJapHD) [inline minecraft_servers/gamerherz/manifest.json:25]: JSON key in lowercase

File: minecraft_servers/gamerherz/manifest.json [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+      "name": "Schach", 
+      "command": "/quickjoin Schach-1",
+      "versions": "1.8<*"
+    },
+    "Duels 1.16": {
```
Comment (DoJapHD) [inline minecraft_servers/gamerherz/manifest.json:30]: JSON key in lowercase

File: minecraft_servers/gamerherz/manifest.json [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+      "name": "Duels 1.16", 
+      "command": "/quickjoin Duels_16-1",
+      "versions": "1.16<*"
+    },
+    "Schach": {
```
Comment (DoJapHD) [inline minecraft_servers/gamerherz/manifest.json:35]: JSON key in lowercase

File: minecraft_servers/gamerherz/manifest.json [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+    "youtube": "https://www.youtube.com/channel/UCoPAhhSl7Y3sek7qXkQCqWw",
+  },
+  "gamemodes": {
+    "Paintball": {
+      "name": "Paintball", 
+      "command": "/quickjoin Paintball-1",
+      "versions": "1.8<*"
+    },
+    "Duels 1.8": {
+      "name": "Schach",
```
Comment (DoJapHD) [inline minecraft_servers/gamerherz/manifest.json:26]: Entry doesn't match to JSON key

File: minecraft_servers/gamerherz/manifest.json [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+      "command": "/quickjoin Schach-1",
+      "versions": "1.8<*"
+    },
+    "Duels 1.16": {
+      "name": "Duels 1.16", 
+      "command": "/quickjoin Duels_16-1",
+      "versions": "1.16<*"
+    },
+    "Schach": {
+      "name": "Duels 1.8",
```
Comment (DoJapHD) [inline minecraft_servers/gamerherz/manifest.json:36]: Entry doesn't match to JSON key

File: minecraft_servers/gamerherz/background.png
Comment (DoJapHD) [inline minecraft_servers/gamerherz/background.png:1]: Still not compressed

File: minecraft_servers/gamerherz/background@2x.png
Comment (DoJapHD) [inline minecraft_servers/gamerherz/background@2x.png:1]: Still not compressed

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/gamerherz/manifest.json (hunk 1 above)
Comment (Nicouschulas) [reply minecraft_servers/gamerherz/manifest.json:13]: Is back online in 2 or 3 days

Comment (Nicouschulas) [reply minecraft_servers/gamerherz/manifest.json:13]: Yeah, and today it's back online. It was planned to be online today and now it's back online

File: minecraft_servers/gamerherz/background.png
Comment (DoJapHD) [inline minecraft_servers/gamerherz/background.png:1]: Compress

File: minecraft_servers/gamerherz/background@2x.png
Comment (DoJapHD) [inline minecraft_servers/gamerherz/background@2x.png:1]: Compress

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by DoJapHD** [comment]
Invalid hDPI logo size! Size is 715x631px, shortest side must not exceed 512px (minecraft_servers/gamerherz/logo@2x.png)
Invalid logo size! Size is 715x631px, shortest side must not exceed 256px (minecraft_servers/gamerherz/logo.png)

**Comment by Nicouschulas** [comment]
> Invalid hDPI logo size! Size is 715x631px, shortest side must not exceed 512px (minecraft_servers/gamerherz/logo@2x.png)
> Invalid logo size! Size is 715x631px, shortest side must not exceed 256px (minecraft_servers/gamerherz/logo.png)

Hello DoJapHD,
I compress this file 3 times and upload it but seemingly it's not uploading/replacing it right. Do you know how I can fix this? 
Thanks, Nicouschulas

**Pull request closed without merging**